src/domain/operation_mode.py

```diff
@@ -120,11 +118,17 @@ def send_email(self, body, img_path):
             smtp_server.quit()
         logger.info("Email notification for %s sent!", img_path)
 
-    def actuate(self):
+    def actuate(self, actuator_list):
         """Method to trigger actuators when enabled"""
-        if actuators:
-            # TODO: implement actuator logic
-            pass
+        # TODO @stefano
+        for actuator in actuator_list:
+            if actuator.enabled:
+                if isinstance(actuator, RoadSignActuator):
+                    actuator.send_command(RoadSignActuator.Commands.display_on)
+                elif isinstance(actuator, FeederActuator):
+                    actuator.send_command(FeederActuator.Commands.open)
+                else:
+                    raise Exception("Unknown actuator type")
```

Comment:
log

Reply:
this is a to-do method; I've inserted some lines just to give you an idea on what the method could do.